spec/controllers/admin/rentals_controller_spec.rb

```diff
@@ -97,6 +97,40 @@
       end
     end
 
+    context "when there is large amount of blank remote accounts" do
+      let!(:blank_remote_accounts) { create_list(:remote_account, 23, account: account) }
+
+      context "when configuration is set to not ignore blank remote accounts" do
+        it "assigns all remote accounts" do
+          index_with_search
+          expect(assigns(:remote_accounts).count).to eq(26)
```

Comment:
maybe something like:
```ruby
expect(assigns(:remote_accounts)).to include(*blank_remote_accounts)
```
to make the spec more resilient if there are some setup changes outside these new contexts.